.github/workflows/vanilla_workflows.yaml

```diff
@@ -46,7 +46,16 @@ jobs:
           REPLICATE_API_TOKEN: ${{ secrets.REPLICATE_API_TOKEN }}
         run: |
           notebooks=$(cat .github/notebook_lists/vanilla_notebooks.txt | tr '\n' ' ')  
-          for notebook in $notebooks; do  
-            echo "Executing $notebook"  
-            jupyter nbconvert --to notebook --execute --inplace $notebook  
+          for notebook in $notebooks; do 
+            echo "Building virtual env"  
```

Comment:
Two questions:
1) Why are you making a virtual environment?
2) Why are you doing it here, not in it's own step?